internal/keypairs/mock/mock.go

```diff
@@ -0,0 +1,41 @@
+package mock
```

Comment:
Please rename this file to `mock_test.go`

Reply:
I couldn't do that because then the structs would not be found. I moved this package into the `internal/testhelpers/` area, as this was done with other mocks like secretsapi. Let me know if you prefer a different package name.